content/docs/next-release-v2/frontend-ui.md

```diff
@@ -66,6 +66,11 @@ An example configuration file (see [complete schema here](https://github.com/jae
     "key": "uniqueId",
     "url": "https://mykibana.com/uniqueId=#{uniqueId}&traceId=#{trace.traceID}",
     "text": "Redirect to kibana to view log"
+  },
+  {
+    "type": "traces",
+    "url": "https://mykibana.com/_dashboards/app/discover#/?_g=(time:(from:'#{startTime | epoch_micros_to_date_iso}',to:'#{endTime | epoch_micros_to_date_iso}'))",
```

Comment:
can we simplify this example? It does not need to be the exact Kibana query, and in the current form it's pretty difficult to read. Something like:
```suggestion
    "url": "my-logs.server/?from=#{startTime | add -60000000 | epoch_micros_to_date_iso}&to=#{endTime | add 60000000 | epoch_micros_to_date_iso}'",
```